src/components/list/CouponList/CouponList.tsx

```diff
@@ -0,0 +1,75 @@
+import styled from '@emotion/styled';
+import CouponItem from '../../box/CouponItem/CouponItem';
+import useCouponFetch from '../../../hooks/useCouponFetch';
+import Button from '../../common/Button/Button';
+import { useConfirmModal } from '../../../hooks/useConfirmModal';
+import LoadingSpinner from '../../common/LoadingSpinner/LoadingSpinner';
+
+const CouponList = () => {
+  const { allCoupon, addCouponAPI, isFetching } = useCouponFetch();
+  const { openModal } = useConfirmModal();
+
+  if (isFetching) {
+    return <LoadingSpinner />;
+  }
+  const addCoupon = async (couponId: number) => {
+    addCouponAPI({ id: couponId });
+  };
```

Comment:
```suggestion
  if (isFetching) {
    return <LoadingSpinner />;
  }
  
  const addCoupon = async (couponId: number) => {
    addCouponAPI({ id: couponId });
  };
  
```

전반적으로 코드에 개행이 없는데, 개행을 넣어주세요 :)